### ISSUE TITLE
Prevent Intent from closing

### ISSUE THREAD

**Issue by ali-pedram** [karjooplus/react-native-cafebazaar-intents#2]
I just tested this package and it seems to work fine except when you want to write a comment and click on "ثبت نظر". the keyboard comes on top of the intent view and there is no way to make it go away except click anywhere on the screen, which also closes the intent view and you can't send that comment. the question is how do i prevent the intent view from closing when the user clicks on the screen and only allow dismissing the view with the provided close button ?

**Comment by sijad** [comment]
thanks for using this package.
I'm guessing this is related to cafebazaar itself, do you have any working example that implemented this in java?

**Comment by ali-pedram** [comment]
No, not in Java, also i tested the rating without any comments and the toast said that the rating is sent, but nothing changed on cafe bazaar's side, and no error was thrown. has the API for this changed ? can you check ?

**Comment by sijad** [comment]
I'll look around but no promises.

other issue is also related to cafebazaar too, I think it's because your comment need to be reviewed by cafebazaar admins, it probably takes few hours.
